### ISSUE TITLE
Accessing openAI's GPTs

### ISSUE THREAD

**Issue by hollyebo** [jgraph/drawio#4267]
* [ ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
Learning new knowledge, making visual illustrations of key points, but when I tried out all the drawing tools in GPTs, I wasn't completely satisfied. These tools can only generate simple mind maps, flowcharts, and so on, which is not enough. Because I know drawio is more powerful and rich in features.

**Describe the solution you'd like**
Please connect it to GPTs like Whimsical Diagrams, then automatically convert the knowledge points generated by AI into drawio format, and do secondary editing on drawio or save directly.

**Describe alternatives you've considered**
The best solution for now is to use ChatGPT to generate mermaid code or plantUML code, then open drawio and import the code, but it's not very efficient.

**Additional context**
比如 Whimsical Diagrams
![image](https://github.com/jgraph/drawio/assets/38182635/07be9d8e-ecc9-4a94-8f5b-a80660b09c79)
![image](https://github.com/jgraph/drawio/assets/38182635/782a9ffa-7a05-4a51-96ae-21edaaf125a6)

**Comment by davidjgraph** [comment]
And the correct solution doesn't solve the issue for you https://www.diagrams.net/blog/smart-diagram-generation ?

**Comment by hollyebo** [comment]
> And the correct solution doesn't solve the issue for you https://www.diagrams.net/blog/smart-diagram-generation ?
No, he can handle it. But I need higher efficiency. I need to visualize and simplify new knowledge. I want to be able to generate design images that match the contents I need just by describing the knowledge points. This way, I can learn and understand new knowledge.

**Comment by davidjgraph** [comment]
If you have time, could you please give an example of using our current mechanism against the flow you're suggesting to implement and what the advantage of the new flow is? Thanks.
